src/export.rs

```diff
@@ -0,0 +1,25 @@
+use crate::utils::{derive_account_with_index, DEFAULT_WALLETS_VAULT_PATH};
+use anyhow::Result;
+use std::{io::Write, path::PathBuf};
+use termion::screen::AlternateScreen;
+
+pub(crate) fn export_account(path: Option<String>, account_index: usize) -> Result<()> {
+    let wallet_path = match &path {
+        Some(path) => PathBuf::from(path),
+        None => home::home_dir().unwrap().join(DEFAULT_WALLETS_VAULT_PATH),
```

Comment:
It looks like this path gets used quite a bit, perhaps we could have some `default_home_wallets_vault_path` function for it or similar?

A more idiomatic approach to retrieving the `wallet_path` might look like:

```rust
let wallet_path = path
    .map(PathBuf::from)
    .unwrap_or_else(default_home_wallets_vault_path);
```

Reply:
Yep, this looks much better thanks! Since I have couple of wallet path usage I am thinking of cleaning all of them together with a future PR if that sounds good to you.